# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../minit/sysvinit-shutdown.diff
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

In the evil case we have sysvinit, don't overwrite it's shutdown

--- ./Makefile.orig	2006-09-27 12:41:39.000000000 -0400
+++ ./Makefile	2006-09-27 12:43:01.000000000 -0400
@@ -78,7 +78,7 @@
 	install minit pidfilehack $(DESTDIR)/sbin
 	install write_proc hard-reboot minit-update $(DESTDIR)/sbin
 	install msvc serdo $(DESTDIR)/bin
-	install -m 4750 shutdown $(DESTDIR)/sbin
+	install -m 4750 shutdown $(DESTDIR)/sbin/minit-shutdown
 	test -f $(DESTDIR)/sbin/init || ln $(DESTDIR)/sbin/minit $(DESTDIR)/sbin/init
 	install -m 644 hard-reboot.8 minit-list.8 minit-shutdown.8 minit-update.8 minit.8 msvc.8 pidfilehack.8 serdo.8 $(DESTDIR)$(MANDIR)/man8
 
